jwst/regtest/conftest.py

```diff
@@ -196,6 +209,18 @@ def rtdata_module(artifactory_repos, envopt, request, jail):
     return _rtdata_fixture_implementation(artifactory_repos, envopt, request)
 
 
+def _sdpdata_fixture_implementation(artifactory_repos, envopt, request):
+    """Provides the RemoteResource class"""
+    inputs_root, results_root = artifactory_repos
+    return SDPPoolsSource(env=envopt, inputs_root=inputs_root,
+                          results_root=results_root)
```

Comment:
The generic part of okify-ing files vs. folder is a generic part of `RegtestData`. 

However, the need for `SDPPoolsSource` is because of the need to use a common set of data between both test construction, see `conftest.pytest_generatetests`, and the test itself. There may be way of generalizing that but that will take some more pytest-foo. Regardless, I'll update some docstrings to make this clear. And maybe lightning will strike.